content/test/unit/service/validations/Validations.spec.ts

```diff
@@ -348,16 +348,56 @@ describe('Validations', function () {
       await assertNoErrors(result)
     })
   })
+
+  describe('IFPS hashing', () => {
+    it(`when an entity's id is not an ipfs hash, then it fails`, async () => {
```

Comment:
Should we test also the older hashing algorithm so it fails?

Reply:
Done!